css/index.css

```diff
@@ -11,25 +11,79 @@ body{
     margin: auto;
     text-align: center;
 }
+
+.navbar{
+    font-weight: bold;
+    display:flex;
+    justify-content: center;
+    font-size: 25px;
+    border-bottom: 25px solid white;  
+}
+a{
+    margin-left: auto;
+    margin-right: auto;
+    text-decoration: none;
+    color: rgb(54, 6, 6);
+}
```

Comment:
currently, a user has no way of telling that the nav section provides links/navigation to other sections. Consider adding a css selector to enhance user experience:
<!-- + text + -->
.navbar a:hover {
       color: red;
}